### DISCUSSION TITLE
How to change the 7860 port to something else?

### DISCUSSION THREAD

**Discussion by Zyin055** [bmaltais/kohya_ss#109] [Q&A] [answered]
Port 7860 is also the default port used by Automatic1111 Web UI. I have a userscript set to load on 127.0.0.1:7860 and it is also being loaded for this since their URLs are identical. How can I change the port being used?

**Comment by Odawgthat** [comment]
FR, I don't like it being on the same port as A1111, ill do some research

**Comment by Zyin055** [comment, marked as answer]
A recent update added the `--server_port XXXX` launch argument.

**Reply by bmaltais** [reply]
Sorry guys. Been too busy with stuff to monitor all discussions and issues... But yes, you can now specify the server port... You can also specify --inbrowser to directly to start the UI in the browser
